### PR TITLE
Add Yash Wankhade to the list

### PR DESCRIPTION
**Pull request by yashwankhade**

### CODE REVIEW

File: public/data/yashwankhade.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/CodingisCoool",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (Panquesito7) [inline public/data/yashwankhade.json:18]:
```suggestion
```

File: public/data/yashwankhade.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Yash Wankhade",
+    "bio": "Learning Web dev || Open Source Enthusiast",
```
Comment (Panquesito7) [inline public/data/yashwankhade.json:3]:
```suggestion
    "bio": "Learning Web Development || Open Source Enthusiast",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/yashwankhade.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/yashwankhade",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/CodingisCoool",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/yashwankhade.json:17]:
```suggestion
}

```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by yashwankhade** [comment]
Thank you :)

**Pull request merged**